themes/vocabulary_theme/templates/blocks/hero.html

```diff
@@ -0,0 +1,23 @@
+<section>
+  <h1 class="column is-9 is-paddingless">
```

Comment:
same here. maybe `hero-title` could be a good option

Reply:
By name, you mean adding an appropriate class right?

Reply:
yeah, that's right

Reply:
There's a `.hero-description` class but no `.hero-title` thi may see a little bit inconsistent in terms of the markup format, also I think it would be better if the title is placed inside a column and not "being" the column space because it makes sense understand the column as a container and inside the container, the elements should be placed